raspberry/bzzz/read_sbus/read_sbus_from_GPIO_receiver.py

```diff
@@ -86,7 +116,14 @@ def get_radio_data_parse_and_send_to_ESP(self,
                                              force_send_fake_data=False,
                                              fake_data="",
                                              over_write_throttle_ref_to=-1):
-        """Read the radio data, process it, format it into a string, and send it via UART.
+        """
```

Comment:
If I remember well, `fake_data` and `force_send_fake_data` were introduced for testing/debugging purposes. If they don't serve a functional purpose, I believe we should remove them. In any case, this can be done in a different PR since this is about documentation only.

Reply:
Yea that's what they where used for, so will we merge this PR and maybe leave issue #118 open and try and solve issues #109 and #179 ?